eng/common/scripts/Helpers/ApiView-Helpers.ps1

```diff
@@ -117,3 +117,106 @@ function Process-ReviewStatusCode($statusCode, $packageName, $apiApprovalStatus,
   $packageNameStatus.IsApproved = $packageNameApproved
   $packageNameStatus.Details = $packageNameApprovalDetails
 }
+
+function Set-ApiViewCommentForRelatedIssues {
+  param (
+    [Parameter(Mandatory = $true)]
+    [string]$HeadCommitish,
+    [string]$APIViewHost = "https://apiview.dev",
+    [ValidateNotNullOrEmpty()]
+    [Parameter(Mandatory = $true)]
+    $AuthToken
+  )
+  . ${PSScriptRoot}\..\common.ps1
+  $issuesForCommit = $null
+  try {
+    $issuesForCommit = Search-GitHubIssues -CommitHash $HeadCommitish
+    if ($issuesForCommit.items.Count -eq 0) {
+      LogError "No issues found for commit: $HeadCommitish"
+      exit 1
+    }
+  } catch {
+    LogError "No issues found for commit: $HeadCommitish"
+    exit 1
+  }
+  $issuesForCommit.items | ForEach-Object {
+    $urlParts = $_.url -split "/"
+    Set-ApiViewCommentForPR -RepoOwner $urlParts[4] -RepoName $urlParts[5] -PrNumber $urlParts[7] -HeadCommitish $HeadCommitish -APIViewHost $APIViewHost -AuthToken $AuthToken
+  }
+}
+
+function Set-ApiViewCommentForPR {
+  param (
+    [Parameter(Mandatory = $true)]
+    [string]$RepoOwner,
+    [Parameter(Mandatory = $true)]
+    [string]$RepoName,
+    [Parameter(Mandatory = $true)]
+    [string]$PrNumber,
+    [Parameter(Mandatory = $true)]
+    [string]$HeadCommitish,
+    [Parameter(Mandatory = $true)]
+    [string]$APIViewHost,
+    [ValidateNotNullOrEmpty()]
+    [Parameter(Mandatory = $true)]
+    $AuthToken
+  )
+  $repoFullName = "$RepoOwner/$RepoName"
+  $apiviewEndpoint = "$APIViewHost/api/pullrequests?pullRequestNumber=$PrNumber&repoName=$repoFullName&commitSHA=$HeadCommitish"
+  LogDebug "Get APIView information for PR using endpoint: $apiviewEndpoint"
+
+  $commentText = @()
+  $commentText += "## API Change Check"
+  try {
+    $response = Invoke-RestMethod -Uri $apiviewEndpoint -Method Get -MaximumRetryCount 3
+    if ($response.Count -eq 0) {
+      LogWarning "API changes are not detected in this pull request."
+      $commentText += ""
+      $commentText += "API changes are not detected in this pull request."
+    }
+    else {
+      LogSuccess "APIView identified API level changes in this PR and created $($response.Count) API reviews"
+      $commentText += ""
+      $commentText += "APIView identified API level changes in this PR and created the following API reviews"
+      $commentText += ""
+      $commentText += "| Language | APIView |"
```

Comment:
Do we want to include Language for sdk repos where it is always just want language?

Reply:
I think it will be best if the structure of API View comments is similar across repositories. But I can condition the formatting on the repo name using `azure-rest-api-specs` repo name.

Reply:
Also please rename the header from APIView to `API review` or `Package`. APIView is the tool.